<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterceptPage</class>
 <widget class="QWidget" name="InterceptPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QWidget" name="crawlerToolbar" native="true">
      <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
        </sizepolicy>
      </property>
      <property name="maximumSize">
        <size>
        <width>16777215</width>
        <height>20</height>
        </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutIntercept">
        <property name="leftMargin">
        <number>0</number>
        </property>
        <property name="topMargin">
        <number>0</number>
        </property>
        <property name="rightMargin">
        <number>0</number>
        </property>
        <property name="bottomMargin">
        <number>0</number>
        </property>
        <item>
        <widget class="QLabel" name="label">
          <property name="font">
          <font>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
          </font>
          </property>
          <property name="text">
          <string>INTERCEPT</string>
          </property>
        </widget>
        </item>
        <item>
        <spacer name="horizontalSpacerIntercept">
          <property name="orientation">
          <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
          <size>
            <width>158</width>
            <height>20</height>
          </size>
          </property>
        </spacer>
        </item>
      </layout>
      </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="interceptTitle">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Intercepted Request:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="forwardButton">
           <property name="text">
            <string>Forward</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="forwardInterceptButton">
           <property name="text">
            <string>Forward + Intercept Response</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dropButton">
           <property name="text">
            <string>Drop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="enabledButton">
           <property name="text">
            <string>Enable Intercept</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="interceptTabs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="headersTab">
        <attribute name="title">
         <string>Headers</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="headersText"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="bodyTab">
        <attribute name="title">
         <string>Body</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="bodyText"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
